src/app.rs

```diff
@@ -871,3 +876,58 @@ impl eframe::App for Damus {
         render_damus(self, ctx);
     }
 }
+
+fn set_demo(
+    data_path: &DataPath,
+    ndb: &Ndb,
+    accounts: &mut Accounts,
+    columns: &mut Columns,
+    unk_ids: &mut UnknownIds,
+) {
+    let demo_pubkey =
+        Pubkey::from_hex("82341f882b6eabcd2ba7f1ef90aad961cf074af15b9ef44a09f9d2a8fbfbe6a2")
+            .unwrap();
+    {
+        let txn = Transaction::new(ndb).expect("txn");
+        accounts
+            .add_account(Keypair::only_pubkey(demo_pubkey))
+            .process_action(unk_ids, ndb, &txn);
+        accounts.select_account(0);
+    }
+
+    columns.add_column(Column::new(vec![Route::Accounts(AccountsRoute::Accounts)]));
+    let timeline = TimelineKind::contact_list(timeline::PubkeySource::Explicit(demo_pubkey))
+        .into_timeline(ndb, Some(demo_pubkey.bytes()));
+
+    if let Some(timeline) = timeline {
+        columns.add_new_timeline_column(timeline);
+    }
+    storage::save_columns(data_path, columns.as_serializable_columns());
+}
+
+fn handle_first_app_open(
+    data_path: &DataPath,
+    ndb: &Ndb,
+    accounts: &mut Accounts,
+    columns: &mut Columns,
+    unk_ids: &mut UnknownIds,
+) {
+    let file_name = String::from("opened");
+
+    if Directory::new(data_path.path(DataPathType::Setting))
+        .get_file(file_name.clone())
+        .is_err()
+    {
```

Comment:
we don't really need this, we can just use the same mechanism we had before in Damus::new when we have no columns:

```rust
        if columns.columns().is_empty() {
            if accounts.get_accounts().is_empty() {
                // initial column logic here
                columns.add_column(Column::new(vec![
                    Route::AddColumn(AddColumnRoute::Base),
                    Route::Accounts(AccountsRoute::AddAccount),
                ]));
            } else {
                columns.new_column_picker();
            }
        }
```

Reply:
In my opinion, the user should be able to clear all accounts. If they want zero accounts but there's a jack account that mysteriously pops up on every app open I can imagine that to be infuriating. So that's why I came up with this mechanism to only populate the demo account and columns on the first start up and give the user the opportunity to do what they desire after

Reply:
this wouldn't popup on every app open, this only shows up the only time there are empty columns, which only happens the first time you open the app.

the confusing thing was this branch, when I opened the app and it added a new account column and jack column to my existing deck setup.

Reply:
in that conception, where you're using the columns.is_empty for inserting the "demo" configuration: for the user that opens the app for the first time, deletes the jack account, deletes all columns, then closes and opens the app again, the jack account will be added again as well as the demo columns. The user wanted everything to be deleted, but on app open it re-added everything. That's what I was referring to on "every app open". That user doesn't want the demo configuration anymore and doesn't want columns.

But this might not be something any actual user would care about...

> the confusing thing was this branch, when I opened the app and it added a new account column and jack column to my existing deck setup.

not intended, easy fix

Reply:
If that's not a scenario we care about, I will implement it the way you suggested. Do you still want the jack account to be part of the "demo" configuration?

Reply:
> for the user that opens the app for the first time, deletes the jack account, deletes all columns, then closes and opens the app again, the jack account will be added again as well as the demo columns

this is a weird edge case, why do we care about users who want 0 columns, who are these users and why would they want that?

Reply:
> If that's not a scenario we care about, I will implement it the way you suggested. Do you still want the jack account to be part of the "demo" configuration?

we have a different account now, use this one:

npub14fenpq0y7rmem4psy0vfsvn9tyljksdf3pn3el808ayfhyddj0lqkpmcx3